docs/docs/getting-started/first-steps-local.md

```diff
@@ -4,19 +4,25 @@
 With the `constellation mini` command, you can deploy and test Constellation locally without a cloud subscription. This mode is called MiniConstellation. Conceptually, MiniConstellation is similar to [MicroK8s](https://microk8s.io/), [K3s](https://k3s.io/), and [minikube](https://minikube.sigs.k8s.io/docs/).
 <!-- vale on -->
 
-MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node. 
+MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node.
 
 :::info
 
-MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support. 
+MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support.
 For attestation, MiniConstellation currently uses the software-based vTPM provided by KVM/QEMU.
 
 :::
 
+:::caution
+
+MiniConstellation has specific soft- and hardware requirements such as a Linux OS running on an x86-64 CPU. Pay attention to all [prerequisites](#prerequisites) when setting up.
```

Comment:
IMO this is not a reference to a heading (like "see Prerequisites"), but a normal sentence where a link has been inserted. Just commenting on this because I suggested to change it the other way round earlier. Not a too strong opinion though in case you both disagree.

Reply:
Ah I didn't see that comment of yours. I also don't have strong opnions here. 

Your argumentation makes perfect sense to me.